src/apps/front-office/account/pages/Login/LoginPage/LoginPage.tsx

```diff
@@ -19,65 +17,65 @@ export default function LoginPage() {
-                required
-                minLength={14}
+        <div className="min-h-screen flex justify-center items-center">
+          <div className="w-full max-w-[900px] flex flex-col lg:flex-row bg-white rounded-lg overflow-hidden">
+            {/* Image (hidden on small screens) */}
+            <div className="hidden lg:block w-1/2">
+              <img
+                className="h-full w-full object-contain rounded-[15px]"
+                src="/images/login/login-1.png"
+                alt="Woman working on laptop"
               />
-              <PasswordInput
-                name="password"
-                className="w-full px-4 py-2 border border-rateColor rounded-lg focus:outline-none focus:border-priceNowColor h-16"
-                placeholder={trans("password")}
-                required
-                minLength={6}
-                maxLength={16}
-              />
-              <div className="flex items-center justify-between !mb-[50px]">
-                <label className="flex items-center text-sm ">
-                  <CheckBoxInput name="rememberMe" />
-                  <span className="text-sm !ml-2">{trans("rememberMe")}</span>
-                </label>
+            </div>
+
+            {/* Form */}
+            <div className="w-full lg:w-1/2 p-8">
+              <h1 className="text-6xl text-mainColor font-semibold mb-4">
+                {trans("login")}
+              </h1>
+              <p className="text-base text-[#7E7E7E] font-normal leading-6 mb-5">
+                {trans("register")}
                 <Link
-                  to={URLS.auth.forgetPassword}
-                  className="text-sm cursor-pointer text-rateColor font-semibold duration-300">
-                  {trans("forgetPassword")}
+                  to={`/register`}
+                  className="cursor-pointer text-priceNowColor font-semibold duration-300 hover:text-[#FDC040] ml-1">
+                  {trans("createHere")}
                 </Link>
-              </div>
-              <Button className="bg-mainColor hover:bg-primary text-white !px-10 !py-[15px]">
-                {trans("loginButton")}
-              </Button>
-            </Form>
+              </p>
+
+              <Form className="space-y-4" onSubmit={handleSubmit}>
+                <EmailInput
+                  name="email"
+                  className="w-full px-4 py-2 border border-rateColor rounded-lg focus:outline-none focus:border-priceNowColor h-16"
+                  placeholder={trans("email")}
+                  required
+                  minLength={14}
+                />
+                <PasswordInput
+                  name="password"
+                  className="w-full px-4 py-2 border border-rateColor rounded-lg focus:outline-none focus:border-priceNowColor h-16"
+                  placeholder={trans("password")}
+                  required
+                  minLength={6}
+                  maxLength={16}
```

Comment:
There should be no maxlength for password I think, no?

Reply:
fixed in the next PR `wishlist-API`